pkg/reconciler/v1alpha1/trigger/trigger.go

```diff
@@ -67,19 +74,23 @@ type reconciler struct {
 	dynamicClient dynamic.Interface
 	recorder      record.EventRecorder
 
+	triggersLock sync.RWMutex
+	triggers     map[string]map[reconcile.Request]bool
```

Comment:
I think we can make the key the namespaced name of the Broker, that way we only reconcile triggers that correspond to the specific broker.

`map[types.NamespacedName]map[reconcile.Request]bool`

And possibly swap out bool for struct{}.

Reply:
Done